.env.telos

```diff
@@ -41,17 +41,27 @@ HAPI_SYNC_PRODUCER_INFO_INTERVAL=1
 HAPI_SYNC_PRODUCER_CPU_INTERVAL=6
 HAPI_SYNC_EXCHANGE_RATE=86400
 HAPI_EOS_EXCHANGE_RATE_API=https://api.coingecko.com/api/v3/simple/price?ids=telos&vs_currencies=usd
+HAPI_SYNC_SCHEDULE_HISTORY_INTERVAL=86400
 HAPI_COINGECKO_API_TOKEN_ID=telos
 HAPI_REWARDS_TOKEN=TLOS
 HAPI_SYNC_STATS_INTERVAL=60
+HAPI_EOS_STATE_HISTORY_PLUGIN_ENDPOINT=
+HAPI_EOS_GET_FAUCET_ACCOUNT=
+HAPI_EOS_FAUCET_ACCOUNT=
+HAPI_EOS_FAUCET_ACCOUNT_PASSWORD=
+HAPI_EOS_MECHANICS_INCLUDE_TRANSACTION=
+GOOGLE_APPLICATION_CREDENTIALS=
+HAPI_RE_CAPTCHA_PROJECT_ID=
+HAPI_PUBLIC_RE_CAPTCHA_KEY=
+HAPI_CREATE_ACCOUNT_ACTION_NAME=
 
 #webapp
 PORT=3000
-REACT_APP_TAG=dev
+REACT_APP_VERSION=dev
 REACT_APP_NAME=$npm_package_name
 REACT_APP_TITLE=Telos Mainnet Network Dashboard
 REACT_APP_DEFAULT_PRODUCER_LOGO=https://ipfs.eosio.cr/ipfs/QmX5ENinjMXANryNJWTEWBzxtojtaWzkW1L27YRorrcpcJ
```

Comment:
we are no longer using IPFS for images 

lets set this to : https://eosio.online/images/telos.jpg